### ISSUE TITLE
Add ability to check when user was last seen in chat.

### ISSUE THREAD

**Issue by JakubNvk** [mrshu/brutal-plugins#95]
Example:
```
<user> !last_seen JakubN
<bot> JakubN was last seen on <date> <time>.
```
This would work for any event made by the user.

**Comment by mrshu** [comment]
@JakubNvk I'd suggest that the command could be called `seen` and it should also remember the type of event created by this user (for instance `message`, `join`, `part` or `nickchange`)

**Comment by Adman** [comment]
@JakubNvk :+1:

**Comment by JakubNvk** [comment]
@mrshu there are some issues with `seen`, so Ill go with `last_seen` for now.

**Comment by mrshu** [comment]
@JakubNvk is this already done?

cc @Adman

**Comment by JakubNvk** [comment]
@mrshu yes.
Fixed: #97
